src/ivc/public_params.rs

```diff
@@ -350,15 +367,15 @@ mod pp_test {
     type RandomOracleConstant<const T: usize, const RATE: usize, F> =
         <RandomOracle<T, RATE> as ROPair<F>>::Args;
 
-    #[test]
+    #[test_log::test]
```

Comment:
what are the pros to use "test_log" instead of built-in test macro?  What are typical commands to use it?

Reply:
The only purpose is to switch on the logger